sematic/ui/src/pipelines/PipelineIndex.tsx

```diff
@@ -86,9 +86,9 @@ function PipelineRow(props: { run: Run }) {
       <TableRow key={id}>
         <TableCell key="name" data-cy={"pipeline-row"}>
           <Box sx={{ mb: 3 }}>
-            <Link href={"/pipelines/" + calculator_path} underline="hover">
+            <MuiRouterLink href={"/pipelines/" + calculator_path} underline="hover">
```

Comment:
Aren't there cases when we want to perform a full page reload, such as switching the pipeline, in order to trigger the callbacks that refresh context variables, such as the run ID?

Reply:
Yeah, the react-router `<Link>` will change the URL with history API without actually reloading the page, after that, the router's routing logic will be triggered.